fs/dtype_test.go

```diff
@@ -0,0 +1,18 @@
+package fs
+
+import (
+	"testing"
+
+	"github.com/containerd/containerd/testutil"
+)
+
+func TestRequiresRootNOP(t *testing.T) {
+
+	// This is a dummy test case that exist to call
```

Comment:
without this running "gmake root-coverage" fails on non-linux platforms with the following "go test" error:

    flag provided but not defined: -test.root

the issue is that the makefile determines which tests to run (TEST_REQUIRES_ROOT_PACKAGES) by doing "git grep -l testutil.RequiresRoot".  in the fs package the only tests that references testutil.RequiresRoot is in dtype_linux_test.go, which is only built on linux.  but grep is not go build tag aware, so when the makefile root-coverage target runs "go test ... -test.root" on non-linux platforms, it incorrectly includes the fs package on the command line.  this fails on non-linux platforms with the error message above because there are no fs package tests that reference testutil.RequiresRoot.  i didn't see any easy way to fix the makefile to be go build tag aware, so to work around the problem i added this file to create a reference to testutil.RequiresRoot so that the "go test" commands runs successfully.

i'll try expanding the comment in this file to explain this more clearly.

if there's a better way to solve this problem then please let me know.